packages/paste-core/primitives/modal-dialog/stories/index.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import {storiesOf} from '@storybook/react';
+import {withKnobs} from '@storybook/addon-knobs';
+import styled from '@emotion/styled';
+import {Text} from '@twilio-paste/text';
+import {Button} from '@twilio-paste/button';
+import {ModalDialogOverlay, ModalDialogContent} from '../src';
+
+const StyledModalDialogOverlay = styled(ModalDialogOverlay)({
+  position: 'fixed',
+  top: 0,
+  right: 0,
+  bottom: 0,
+  left: 0,
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  background: 'rgba(0, 0, 0, 0.7)',
+});
+
+const StyledModalDialogContent = styled(ModalDialogContent)({
+  width: '100%',
+  maxWidth: '560px',
+  maxHeight: 'calc(100% - 60px)',
+  background: '#f4f5f6',
+  borderRadius: '5px',
+});
+
+interface BasicModalDialogProps {
+  isOpen: boolean;
+  handleClose: () => void;
+}
+
+const BasicModalDialog: React.FC<BasicModalDialogProps> = ({isOpen, handleClose}) => (
```

Comment:
I split this up into a separate component to showcase how the code should be split up.  Lots of examples keep this JSX code inside the other component, but for real applications I think decoupling makes the most sense.